airbyte-integrations/connectors/destination-oracle/src/test-integration/java/io/airbyte/integrations/destination/oracle/NneOracleDestinationAcceptanceTest.java

```diff
@@ -18,7 +20,7 @@
 import java.sql.SQLException;
 import java.util.List;
 import java.util.Map;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
what is this change doing and why is it needed?

Reply:
our main test suite is running on the Junit5 version, but the `Test` annotation was added from the Junit4 version, that's why it was not triggered by `/test` in the past, this update fixes the described issue